integration/src/integration/java/pl/allegro/tech/hermes/integration/ZookeeperSecureTest.java

```diff
@@ -0,0 +1,60 @@
+package pl.allegro.tech.hermes.integration;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.RetryOneTime;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.zookeeper.KeeperException;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import spock.lang.Shared;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+
+
+public class ZookeeperSecureTest extends IntegrationTest {
```

Comment:
I understand that you added this test for demonstration purposes but do we want to merge it to master branch?